tests/base/print_partexa_header.cc

```diff
@@ -0,0 +1,36 @@
+// ---------------------------------------------------------------------
+//
+// PartExa - A Particle Library for the Exa-Scale
+//
+// Copyright (C) 2021 by the PartExa authors
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+//
+// ---------------------------------------------------------------------
+
+#include <partexa/base/print_infos.h>
+
+#include "../tests.h"
+
+int
+main()
+{
+  dealii::initlog();
+
+  dealii::ConditionalOStream pcout(dealii::deallog.get_file_stream(), true);
+
+  PartExa::print_partexa_header(pcout);
+
+  return 0;
+}
```

Comment:
@peterrum: Do you prefer to make use of `using namespace dealii;` and `using namespace PartExa` here?

Reply:
I don't really care. Personally, I prefer to be explicit regarding namespaces (i.e. not use `using namespace dealii;`) - this is what we try to do in hyper.deal (except in the test folder^^). The PartExa project is still at the beginning so banning the usage of `using namespace dealii;` is still possible without too much headache. `using namespace PartExa` is reasonable; these are the tests of PartExa^^

Reply:
I would aim to not use `using namespace dealii;`. For example, I would try to avoid this when writing ExaDG once more. I think the code is overall easier to understand if one immediately sees where a certain functionality originates from.

Reply:
Thanks, its good to hear that both of you confirm that. So let's do that, and don't use `using namespace dealii;` in PartExa. I add that to the contribution guidelines as soon as I found the time to set up a wiki.